extensions/amp-ad/0.1/amp-ad.css

```diff
@@ -24,7 +24,32 @@ amp-ad iframe {
   padding: 0 !important;
 }
 
-.amp-ad-default-fallback {
+.amp-ad-default-display {
   /* TODO(zhouyx): Confirm on the background-color value */
   background-color: #d3d3d3;
 }
+
+.-amp-ad-default-holder {
+  position: absolute;
+  left: 0;
+  right: 0;
+  top: 0;
+  bottom: 0;
+  display: flex;
```

Comment:
I add this extra holder wrapper because we will override all placeholder to `display:block` later. Should I put `display: flex !important` here and apply the style to default placeholder, or should I keep it `<placeholder><div display:flex><tag></tag></div></placeholder>` don't have a preference.